numba_dpex/kernel_api/launcher.py

```diff
@@ -112,17 +112,26 @@ def _ndrange_kernel_launcher(kernel_fn, index_range, *kernel_args):
             )
 
 
-def call_kernel(kernel_fn, index_range, *kernel_args):
+def call_kernel(kernel_fn, index_range: Range | NdRange, *kernel_args):
```

Comment:
```suggestion
def call_kernel(kernel_fn, index_range: Union[Range, NdRange], *kernel_args):
```
Please use Union for older python support.

Reply:
Done